as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/AbstractTest.java

```diff
@@ -36,4 +37,22 @@ public static void deleteRuntimes() {
 		runtimePreferencePage.removeAllRuntimes();
 		preferences.ok();
 	}
+    
+    public static void addJRE(String name, String path) {
```

Comment:
Can be deleted. (just a suggestion)

Reply:
If is not used elsewhere.

Reply:
this is used in ServerRuntimesTest, I will rewrite it in different PR.

Reply:
goooood